ui/src/scss/_patterns_double-row.scss

```diff
@@ -0,0 +1,53 @@
+@mixin maas-double-row {
+  $checkbox-space: 1rem + $sph-inner;
+  $icon-space: map-get($icon-sizes, default) + $sph-inner--small;
+
+  .p-double-row {
+    overflow: visible;
+
+    %p-double-row__icon-container {
+      display: block;
+      float: left;
```

Comment:
We should definitely move away from using floats. They end up requiring a lot of workarounds to get things to line up, so I would suggest using a simple grid for the double row instead. I've quickly written an example here: https://pastebin.canonical.com/p/zdDZyz4WmC/

Note this only makes sense for columns that have an icon or checkbox - the other ones could probably just have a flex column.

Reply:
Excellent, thank you @Caleb-Ellis - that's much simpler and clearer!